pegasus/sites.v3/code.org/views/access_report_skinny_banner.haml

```diff
@@ -0,0 +1,15 @@
+:ruby
+  hoc_mode = DCDO.get('hoc_mode', CDO.default_hoc_mode)
+
+- if hoc_mode != "soon-hoc" && !!DCDO.get('access-report-launch', false)
```

Comment:
hm I think hoc_mode will be soon-hoc for another couple of months (right @kelbyhawn?). @dmcavoy what are your thoughts here? It seems like there are too many banners on /administrators.

Reply:
I think the HOC banner could be moved further down the page since it's been up for awhile. Maybe above the "A comprehensive K-12 curriculum pathway" section?

Reply:
Done!

Reply:
Works for me!